v8/keytab/keytab.go

```diff
@@ -21,7 +22,7 @@ const (
 
 // Keytab struct.
 type Keytab struct {
-	version uint8
+	Version uint8
```

Comment:
What is the need to make this field public?

Reply:
I need to set it to 2. I could keep it non-exported and create the Keytab with another constructor (`NewWithVersion(version uint8) (*Keytab, error)`) or some sort of setter method `(k *Keytab) SetVersion(version uint8) error`. The valid values are 1 or 2.

What do you think?

Reply:
I was thinking it would make sense to set the version to a valid value in the ``New()`` function. How about we update that to set it to 2?
I like to only make things public if they really need to be.